src/esrs/postgres/event.rs

```diff
@@ -18,14 +16,14 @@ pub struct Event {
     pub sequence_number: SequenceNumber,
 }
 
-impl<E: Serialize + DeserializeOwned + Send + Sync> TryInto<StoreEvent<E>> for Event {
+impl<Payload: serde::de::DeserializeOwned> TryInto<StoreEvent<Payload>> for Event {
     type Error = serde_json::Error;
 
-    fn try_into(self) -> Result<StoreEvent<E>, Self::Error> {
+    fn try_into(self) -> Result<StoreEvent<Payload>, Self::Error> {
```

Comment:
I don't know if I like explicit generics. Usual convention is (even in `std`) to call them with single letters - my main gripe being that I can't tell on the fly if `Payload` is a generic or an existing type in the scope.

Reply:
Mmmmh ok, so this concept is valid for the other generics in the library? For example the ones in the `PgStore`? It seems to me that give a generic that explains what are you putting in your trait/struct helps the user to understand what should be put inside of it..

Reply:
@miterst and @matteosister wdyt?

Reply:
Changed to `E`. Anyway `Event` is not exposed to users, so it's ok for me to call it `E` or whatever.